src/control/PlanarArmController.h

```diff
@@ -85,12 +86,19 @@ class PlanarArmController {
 	 * @param targetVel The target x velocity.
 	 * @return The new command, which is the new joint positions.
 	 */
-	void set_x_vel(robot::types::datatime_t currTime, double targetVel) {
+	void set_x_vel(robot::types::datatime_t currTime, double targetVel,
+				   robot::types::DataPoint<navtypes::Vectord<Constants::arm::IK_MOTORS.size()>>
```

Comment:
The classes in the `control` folder (and most other folders) should be completely modular, so they should minimize dependencies on global variables. In this case, we have `N` as a template parameter for the class, which is the number of joints, which we should use instead of `Constants::arm::IK_MOTORS.size()`, so that should be replaced with `N`, and the include of `Constants.h` should be removed